client/my-sites/checkout/checkout-thank-you/domains/domain-mapping-thank-you.jsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+import { ThankYou } from 'calypso/components/thank-you';
+
+import { CALYPSO_CONTACT, SUPPORT_ROOT } from 'calypso/lib/url/support';
+import { Button } from '@automattic/components';
+import domainConnectedSuccess from 'calypso/assets/images/illustrations/domain-connected-success.svg';
+
+import './style.scss';
+
+const DomainMappingThankYou = ( { domain, translate } ) => {
+	return (
```

Comment:
Minor non-blocking note: it may be easier to follow the code if you use the `useTranslate` hook from `i18n-calypso` to access `translate` from within your component. I find that's easier to follow than it being injected via `localize`.
```suggestion
const DomainMappingThankYou = ( { domain } ) => {
	const translate = useTranslate();

	return (
```